src/appMain/hmi_capabilities.json

```diff
@@ -10,169 +10,169 @@
             "displayName": "GENERIC_DISPLAY",
             "textFields": [{
                     "name": "mainField1",
-                    "characterSet": "TYPE2SET",
+                    "characterSet": "UTF_8",
```

Comment:
Should this just be ASCII since "All systems will support at least ASCII"

Reply:
From the proposal https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0202-character-sets.md
> These character sets were chosen because they are the most widely supported character sets and are supported on all Android and iOS devices. As noted in the document, UTF-8 is the preferred character set.

Also, I think setting `UTF_8` as the default would allow us to show the full capabilities as opposed to just using the minimum supported character set `ASCII`.